tools/rosetta/lib/server/server.go

```diff
@@ -41,6 +41,7 @@ type Server struct {
 }
 
 func (h Server) Start() error {
+	fmt.Printf("Rosetta server listening on add %s", h.addr)
```

Comment:
is the server allowed to be chatted and just print out to os.Stdout? If so, is there a way to add a logger to Server and instead use that?

Reply:
Hi! is there any logging service inside the sdk that we can use to avoid these `printf` ?

Reply:
@raynaudoe probably `github.com/tendermint/tendermint/libs/log`